_rules/css-restrict-orientation-b33eff.md

```diff
@@ -0,0 +1,255 @@
+---
+id: b33eff
+name: Page is not restricted to either `landscape` or `portrait` orientation using CSS transform property
+rule_type: atomic
+description: |
+  This rule checks that page content is not restricted to either `landscape` or `portrait` orientation using CSS transform property.
+accessibility_requirements:
+ wcag20:1.3.4: # Orientation
+  forConformance: true
+  failed: not satisfied
+  passed: further testing needed
+  inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to any element that is [visible](#visible) and has a CSS [transform](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) property with any of the below [transformation functions](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function)
+
+- [rotate](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate)
+- [rotate3d](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate3d)
+- [rotateX](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotateX)
+- [rotateY](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotateY)
+- [rotateZ](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotateZ)
+- [matrix](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix)
+- [matrix3d](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix3d)
+
+that are applied conditionally on the [orientation](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation) [media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#Media_features) with a value of `landscape` or `portrait`.
+
+**Note:** These specific [transformation functions](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function) are of interest to this rule as they have the potential to affect the [rotation](https://drafts.csswg.org/css-transforms-2/#Rotate3dDefined) of a given element.
+
+## Expectation 1
+
+In `portrait` orientation, the target element is neither rotated clockwise nor counter clockwise around the Z-axis at an angle corresponding to 90 degrees relative to the position of the element in `landscape` orientation.
+
+## Expectation 2
+
+In `landscape` orientation, the target element is neither rotated clockwise nor counter clockwise around the Z-axis at an angle corresponding to 90 degrees relative to the position of the element in `portrait` orientation.
```

Comment:
I'm only just now realising that these two expectations are equivalent 🙈 Could you sanity check this, @Jym77?

Reply:
Rephrasing the expectation, nevermind!